src/dcd/common/messages.d

```diff
@@ -246,3 +292,51 @@ bool serverIsRunning(bool useTCP, string socketFile, ushort port)
 	else
 		return false;
 }
+
+/// Escapes \n, \t and \ in the string. If single is true \t won't be escaped.
+string escapeTabValue(string s, bool single = false)
```

Comment:
Can this be named something else? If I'm reading this doc comment correctly, it's possible to have this function, named `escapeTabValue`, escape several things and that if you pass `true` then tabs won't even be one of them.